emily/cdk/lib/emily-stack-utils.ts

```diff
@@ -40,9 +41,11 @@ export class EmilyStackUtils {
      */
     public static getAwsAccount(): string {
         this.awsAccount ??= (process.env.AWS_ACCOUNT ?? process.env.CDK_DEFAULT_ACCOUNT);
-        if (this.awsAccount === undefined) {
+        if (this.awsAccount === undefined && this.getStageName() !== Constants.LOCAL_STAGE_NAME) {
             throw new Error('Must define AWS account on either "AWS_ACCOUNT" or "CDK_DEFAULT_ACCOUNT" env variables.');
         }
+        // If the account is undefined and we're letting it slide set the account to 12 "x"s.
```

Comment:
The local environment needs this to be set, but it doesn't matter what it is. Someone who hasn't set up AWS locally shouldn't need to to run the integration tests if it's unnecessary.